lab4/top5airlines.hql

```diff
@@ -31,8 +33,13 @@ LOCATION 'hdfs:///bdpc/hadoop/lab4/airlines';
 
 !echo Running query for Top 5 airlines....;
 
+-- GLC| What about table creation?
```

Comment:
Well, task didn't have any requirements to store result in any way, so I've just printed the result.
But surely it's pretty easy to add something like "create table result as" before this select

Reply:
Oh, actually now it is, I've actually made this homework before the task was published in git, so I didn't know we need to save result :)